src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -23,8 +23,7 @@ export function getStaticProps(context) {
   };
 }
 
-AWS AppSync Apollo Extensions provide a seamless way to connect to your AWS AppSync backend using Apollo client, an open-source GraphQL client. 
-
+AWS AppSync Apollo Extensions provide a seamless way to connect to your AWS AppSync backend using Apollo client, an open-source GraphQL client. The included AWS AppSync authorizers make it simple to apply the correct authorization payloads to your GraphQL operations within the Apollo client. The library allows you to connect directly to your AWS AppSync backend, including AppSync backends managed through Amplify Data.
```

Comment:
This paragraph feels repetitive - I preferred the old intro with the features listed separately.

Reply:
I can split back up. I think it was a mistake but "Features" was at an H2 Level where almost the rest of the documentation was contained under Features. This is primarily why I had removed it in the first place.